From a847d34fb7884de5317a72ea94c2c288b79aa08b Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 30 Oct 2014 22:04:24 +0100
Subject: [PATCH 26/58] prefs: disable translation service

When translation is enabled, the translation manager will request a
list of supported languages from translate.googleapis.com.
As it is Google, we do not want that, at least not without more
explicit user action. One should visit http://translate.google.com/
instead to request translations via Google.
---
 chrome/browser/ui/browser_ui_prefs.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ui/browser_ui_prefs.cc b/chrome/browser/ui/browser_ui_prefs.cc
index 41813f5..e1f7348 100644
--- a/chrome/browser/ui/browser_ui_prefs.cc
+++ b/chrome/browser/ui/browser_ui_prefs.cc
@@ -82,7 +82,7 @@ void RegisterBrowserUserPrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kWebAppCreateInQuickLaunchBar, true);
   registry->RegisterBooleanPref(
       prefs::kEnableTranslate,
-      true,
+      false,
       user_prefs::PrefRegistrySyncable::SYNCABLE_PREF);
   registry->RegisterStringPref(prefs::kCloudPrintEmail, std::string());
   registry->RegisterBooleanPref(prefs::kCloudPrintProxyEnabled, true);
-- 
2.4.3

